.gitlab-ci.yml

```diff
@@ -12,9 +12,9 @@ variables:
   ANSIBLE_LINT_210: 5.4.0
   ANSIBLE_LINT_211: 5.4.0
   ANSIBLE_LINT_212: 5.4.0
-  ANSIBLE_LINT_213: 6.17.2
-  ANSIBLE_LINT_214: 6.17.2
-  ANSIBLE_LINT_215: 6.17.2
+  ANSIBLE_LINT_213: 6.18.0
```

Comment:
Could you pls update to the latest version 6.19.0 ?
https://github.com/ansible/ansible-lint/releases/tag/v6.19.0

Reply:
Should be done :)